src/lib/css/plex-mininav.scss

```
@@ -0,0 +1,168 @@
+@import './variables.scss';
+@import './mixins/_contrast.scss';
+    
+// Mixins estados
+@mixin mininavItemHover {
+    &:hover, &.selected {
+        cursor: pointer;
+        //box-shadow: inset 0 0 0 2px var(--color-base)!important;
+        background-color: var(--color-baseDark)!important;
+        filter: brightness(0.85);
+    }
+}
+
+plex-mininav {
+
+    > section {
+        --mininavWidth: 55px;
+        --mininavHeight: 100vh;
+        --border-color: gray;
+        
+        display: flex;
+        flex-direction: column;
+        width: var(--mininavWidth);
+        height: var(--mininavHeight);
+        background-color: var(--nav-color);
+        transition: width 900ms ease;
+        border-right: solid 1px var(--border-color);
+        color: inherit;
+        
+        nav, ul {
+            display: flex;
+            flex-direction: column;
+        }
+
+        & > nav {
+            --vHeight: 200px;
+            height: calc(100vh - var(--vHeight));
+            position: fixed;
+            overflow-x: hidden;
+
```

```suggestion

    margin-left: -10px;
```
Va como comment, porque lo hablamos en una videollamada: Con eso quedaría centrado horizontalmente respecto al contenedor ppal. Además despega el scroll del contenido en el sidebar.